### PR TITLE
Add configuration option for autoscaler logging verbosity

### PR DESCRIPTION
**Pull request by bison**
Adds configurable verbosity for autoscaler logging.

### CODE REVIEW

File: pkg/operator/config.go [hunk 1]
```diff
@@ -43,6 +43,10 @@ const (
 	// DefaultClusterAutoscalerCloudProvider is the default name for
 	// the CloudProvider beeing used.
 	DefaultClusterAutoscalerCloudProvider = "openshift-machine-api"
+
+	// DefaultClusterAutoscalerVerbosity is the default logging
+	// verbosity level for ClusterAutoscaler deployments.
+	DefaultClusterAutoscalerVerbosity = 2
```
Comment (ingvagabund) [inline pkg/operator/config.go]: at least 3 please :)

File: pkg/operator/config.go [hunk 2]
```diff
@@ -140,5 +149,15 @@ func ConfigFromEnvironment() *Config {
 		config.ClusterAutoscalerNamespace = caNamespace
 	}
 
+	if caVerbosity, ok := os.LookupEnv("CLUSTER_AUTOSCALER_VERBOSITY"); ok {
```
Comment (ingvagabund) [inline pkg/operator/config.go]: +1

File: pkg/operator/config.go (hunk 1 above)
Comment (frobware) [inline pkg/operator/config.go]: I think the consensus on the channel was 1. so, LGTM.

Comment (bison) [reply pkg/operator/config.go]: Done.

**Pull request merged**